### PR TITLE
[Fli iam#729] Reintroduce units of measure

### PR DESCRIPTION
**Pull request by youennmerel**
image

Impacted fields :
- Mr type acquisition
  - MR protocol fields
  - MR protocol metadata fields
- Pet type acquisition
  - Pet protocol fields
- Mr type Dataset
  - Mr dataset data
- Manufacturer model
  - Magnetic field
- Examination
  - Subject weight  (selectable, may be kg or g)
- EEG dataset
  - Sampling frequency

How to test :
- view acquisition of type Mr, Pet
- view EEG datasets
- view manfacturer model
- view / create / edit examination
- view preclinical examination
- create examination & preclinical examination through import

### CODE REVIEW

File: shanoir-ng-front/src/app/preclinical/examination/edit/animal-examination-form.component.ts [hunk 1]
```diff
@@ -400,4 +398,9 @@ export class AnimalExaminationFormComponent extends EntityComponent<Examination>
         this.breadcrumbsService.currentStep.data.examinationNode = node;
     }
 
+    getUnit(key: string) {
+        console.log(key);
```
Comment (jcomedouteau) [inline shanoir-ng-front/src/app/preclinical/examination/edit/animal-examination-form.component.ts:402]: Remove debug logs

File: shanoir-ng-front/src/app/dataset-acquisitions/modality/pet/pet-protocol.component.html [hunk 2]
```diff
@@ -56,11 +56,11 @@
 			<span class="right-col">{{protocol.numberOfSubsets}}</span>
 		</li>
 		<li>
-			<label>Radionuclide half life</label>
+			<label>Radionuclide half life ({{getUnit("SEC")}})</label>
 			<span class="right-col">{{protocol.radionuclideHalfLife}}</span>
 		</li>
 		<li>
-			<label>Radionuclide total dose</label>
+			<label>Radionuclide total dose ({{getUnit("BQ")}})</label>
```
Comment (jcomedouteau) [inline shanoir-ng-front/src/app/dataset-acquisitions/modality/pet/pet-protocol.component.html:63]: MegaBecquerels (Mbq)  ?

File: shanoir-ng-front/src/app/preclinical/examination/edit/animal-examination-form.component.ts [hunk 3]
```diff
@@ -136,7 +145,9 @@ export class AnimalExaminationFormComponent extends EntityComponent<Examination>
     }
 
     initCreate(): Promise<void> {
+        console.log("create !")
```
Comment (jcomedouteau) [inline shanoir-ng-front/src/app/preclinical/examination/edit/animal-examination-form.component.ts:148]: debug logs

File: shanoir-ng-front/src/app/datasets/dataset/mr/dataset.mr.model.ts [hunk 4]
```diff
@@ -2,20 +2,21 @@
...
-import { allOfEnum, capitalsAndUnderscoresToDisplayable } from '../../../utils/app.utils';
-import { Dataset } from '../../shared/dataset.model';
-import { DiffusionGradient } from '../../../dataset-acquisitions/modality/mr/mr-protocol.model';
-import { DatasetType } from '../../shared/dataset-type.model';
+import {Option} from '../../../shared/select/select.component';
+import {allOfEnum, capitalsAndUnderscoresToDisplayable} from '../../../utils/app.utils';
+import {Dataset} from '../../shared/dataset.model';
+import {DiffusionGradient} from '../../../dataset-acquisitions/modality/mr/mr-protocol.model';
+import {DatasetType} from '../../shared/dataset-type.model';
+import {UnitOfMeasure} from "../../../enum/unitofmeasure.enum";
```
Comment (jcomedouteau) [inline shanoir-ng-front/src/app/datasets/dataset/mr/dataset.mr.model.ts:19]: Useless import ?

**Comment by jcomedouteau** [comment]
Ok for me for the tests, we can merge after the debug logs removal :)

**Comment by youennmerel** [comment]
@jcomedouteau all right now ! Thx.

**Review by jcomedouteau — APPROVED** [review]

**Pull request merged**